robMsg/json/de.json

```diff
@@ -134,6 +134,17 @@
 	"LOOP_FOREVER": "Wiederhole unendlich oft",
 	"MENU_ABOUT": "Über das Open Roberta Lab",
 	"MENU_ABOUT_PROJECT": "Über das Open Roberta Projekt",
+	"MENU_SHORTCUT_IMPORT": "Ctrl + I: Programm importieren",
+	"MENU_SHORTCUT_EXPORT": "Ctrl + E: Programm exportieren",
```

Comment:
Please use here "Exportiere Programm" as it is defined in the German translation of the MENU_EXPORT_PROG key